interop-cats/shared/src/main/scala/zio/interop/CQueue.scala

```diff
@@ -29,64 +30,76 @@ final class CQueue[F[+_], -A, +B] private[interop] (
   /**
    * @see [[ZQueue.awaitShutdown]]
    */
-  def awaitShutdown(implicit R: Runtime[Any], F: LiftIO[F]): F[Unit] = toEffect(underlying.awaitShutdown)
+  def awaitShutdown(implicit R: Runtime[Any], F: Async[F]): F[Unit] =
```

Comment:
Could be handled in a follow up but would it be better here to require the capabilities for the creation of the queue and not for each operator? That seems more idiomatic to me from Cats Effect 3 code I have seen.

Reply:
Confirmed :)

Reply:
Definitely if `Async` is involved - you want to minimize the # of places where it's exposed.

Reply:
😃

Reply:
Good point, in this case I mindlessly replaced the constraints to make it compile 😄

Reply:
@adamgfraser there are some deprecated methods on `ZQueue` - what should we do with them?

Reply:
@joroKr21 We should mark them as deprecated for now and then remove them when we release a version that supports ZIO 2.0, which is I think what we are currently doing.

Reply:
@joroKr21 Looks much nicer with your changes now!